TaskModules/powershell/TaskModuleSqlUtility/SqlUtility.ps1

```diff
@@ -0,0 +1,36 @@
+function ConvertFrom-Json20([object] $InputObject){ 
+    add-type -assembly system.web.extensions
```

Comment:
Is the assembly available for any agent service account to load? Can you test with agent service account that is non admin in the box?